plugins/hypervisors/ovm3/src/main/java/com/cloud/hypervisor/ovm3/objects/OvmObject.java

```diff
@@ -172,7 +172,7 @@ public Boolean getSuccess() {
                     if (nodeListFor2.getLength() > 1) {
                         /* Do we need to figure out all the sub elements here and put them in a map? */
                     } else {
-                        String element = nodeListFor.item(index).getTextContent();
+                        String element = nodeListFor.item(index).getNodeValue();
```

Comment:
It looks like other changes are included in this PR that is around move refactoring of package/module from `api.api` to `api`.

Reply:
Yes, there were more changes than the renaming. These other files were not compiling at all. The method `getTextContent` does not exist anymore in XML-APIs. So, this seems like a problem that was ignored during dependencies upgrade. 

The same happens in the code where I had to update the use of `VolumeVO` and `SnapshotVO`. Someone seems to have changed the signature of the method, but did not fix it everywhere.

A similar problem is found for `DirectAgentManagerSimpleImpl.java`, it was not implementing all of the methods from one of its interfaces.

And in `VmSnapshotDaoTest.java`, there is something that would never compile. Someone changed the `VMSnapshotDetailsVO` signature, but did not change all of the places that use it. Therefore, there was an instantiation where the third variable was receiving something called `display`. This `display` is not a variable in that context. It is something auto-generated by Eclipse.